src/main/java/org/example/dao/TodoDAO.java

```diff
@@ -39,33 +40,28 @@ public static void createTaskDB(ImmutableTask task) {
     public static ArrayList<ImmutableTask> getAllTasksDB() {
 
         ConnectionDB connectionDB = new ConnectionDB();
-        PreparedStatement ps = null;
-        ResultSet rs = null;
         ArrayList<ImmutableTask> results = new ArrayList<ImmutableTask>();
+        Byte zero = 0;
 
         try {
             Connection cnx = connectionDB.getConnection();
-
-            try {
-                String query = "SELECT * FROM tasks";
-                ps = cnx.prepareStatement(query);
-                rs = ps.executeQuery();
-
-                while( rs.next()){
-                    ImmutableTask newTask = ImmutableTask.builder()
-                            .id(rs.getInt("id"))
-                            .title(rs.getString("title"))
-                            .description(rs.getString("description"))
-                            .isFinished(rs.getBoolean("isfinished"))
-                            .build();
-
-                    results.add(newTask);
-                }
-                System.out.println("List Tasks");
-
-            } catch (SQLException ex) {
-                System.out.println(ex);
+            DSLContext context = DSL.using(cnx, SQLDialect.MYSQL);
+            /*Result<Record> records = context.select().from(Tasks.TASKS).fetch();*/
+            Result<Record> records = context.fetch("SELECT * FROM tasks");
```

Comment:
Hey @dagibu301 
This is really nice, i was giving a small read to the documentation, have you tried using the  "functions" they provide, that way you don't have to write explicit sequel.

I think you need to use some generators for the tables before using it but it surely make it more fun to work with 😄 
```
// For more info on imports and data types, click on the "Help" icon
// These imports, and possibly others, are implied:
import static org.jooq.impl.DSL.*;
import static com.example.generated.Tables.*;
import org.jooq.*;
import org.jooq.impl.*;

// The "create" reference is an instance of DSLContext

// And with jOOQ...



DSLContext create = DSL.using(connection, dialect);

create.select(AUTHOR.FIRST_NAME, AUTHOR.LAST_NAME, count())
      .from(AUTHOR)
      .join(BOOK).on(BOOK.AUTHOR_ID.eq(AUTHOR.ID))
      .where(BOOK.LANGUAGE.eq("DE"))
      .and(BOOK.PUBLISHED_IN.gt(2008))
      .groupBy(AUTHOR.FIRST_NAME, AUTHOR.LAST_NAME)
      .having(count().gt(5))
      .orderBy(AUTHOR.LAST_NAME.asc().nullsFirst())
      .limit(2)
      .offset(1)
      .forUpdate()
      .fetch();
```

Reply:
That's a great question! yes I tried using the functions but when I did I'm getting an error so in the meantime I wrote it in a explicit way, is the same error I'm having with the update, you can see the code for jooq is commented
You can see in line 49 the use of the function commented

By the way this is the error 
`java: cannot access java.util.concurrent.Flow
  class file for java.util.concurrent.Flow not found`

Reply:
So, I was looking in the internet and it seems that is a java package included from the version 9 and I'm using version 8, but when I upgrade to version 9 I get this error with Jersey, what do you think @mattvv ?

`javax.servlet.ServletException: Servlet.init() for servlet [Jersey Web Application] threw exception
    org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:540)`